ARSIS-Telemetry-GroundControl/telemetry-api/app/caches/user_cache.py

```diff
@@ -0,0 +1,36 @@
+class UserCache:
+    def __init__(self):
+        self.users = {}
+
+    def create_new_user_dict(self):
+        bpm = 100
+        o2 = 100
+        battery = 100
+        latitude = 10
+        longitude = 100
+        altitude = 0
+        heading = 0
+        return {
+            "biometrics": {"bpm": bpm, "o2": o2, "battery": battery},
+            "location": {
+                "latitude": latitude,
+                "longitude": longitude,
+                "altitude": altitude,
+                "heading": heading,
+            },
+        }
+
+    def get_all(self):
+        return self.users
+
+    def get(self, user_id):
```

Comment:
I'm not too python-savy... does anyone know if this has error handling or if it'll just crash?